packages/sdk-v2/src/sdk/goodid/hooks/useCredentials.ts

```diff
@@ -0,0 +1,34 @@
+import { useCallback } from "react";
+import type { VerifiableCredential } from "../types";
+
+// todo: create credential should consider update if already exists
+// todo: add db updateCredential
+// todo: add db deleteCredential
+
+export const useCredentials = (db: any) => {
+  const getActiveCredentials: () => Promise<VerifiableCredential[] | undefined> = useCallback(async () => {
+    try {
+      await db.open();
+      const credentialsList = (await db.credentials.toArray()) as VerifiableCredential[];
+      console.log("credentialsList -->", { credentialsList });
+      return credentialsList;
+    } catch (error) {
+      console.error("Failed to get active credentials:", error);
+    } finally {
+      await db.close();
+    }
+  }, [db]);
+
+  const createCredential = async (credential: VerifiableCredential) => {
```

Comment:
1. call it `storeCredential`. create means issue or generate. we will have 2 dedicated methods to issue location or combined identity certificate taking different inputs
2. also credential is a `type` field value. `certificate` would be better name for the json entities we're issuing. Also they called 'certificates" in the task definitions

Reply:
Changed